x-pack/plugins/security_solution/public/timelines/components/field_renderers/default_renderer/index.test.tsx

```diff
@@ -0,0 +1,133 @@
+      render(
+        <TestProviders>
+          <DefaultFieldRenderer
+            displayCount={5}
+            rowItems={['item1', 'item2']}
+            attrName={'item1'}
+            idPrefix={'prefix-1'}
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('DefaultFieldRendererComponent').textContent).toEqual(
+        'item1,item2 '
+      );
+    });
+
+    test('it should render all items when the item count exactly equals displayCount', () => {
+      render(
+        <TestProviders>
+          <DefaultFieldRenderer
+            displayCount={5}
+            rowItems={['item1', 'item2', 'item3', 'item4', 'item5']}
+            attrName={'item1'}
+            idPrefix={'prefix-1'}
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('DefaultFieldRendererComponent').textContent).toEqual(
+        'item1,item2,item3,item4,item5 '
+      );
+    });
+
+    test('it should render all items up to displayCount and the expected "+ n More" popover anchor text for items greater than displayCount', () => {
+      render(
+        <TestProviders>
+          <DefaultFieldRenderer
+            displayCount={5}
+            rowItems={['item1', 'item2', 'item3', 'item4', 'item5', 'item6', 'item7']}
+            attrName={'item1'}
+            idPrefix={'prefix-1'}
+          />
+        </TestProviders>
+      );
+      expect(screen.getByTestId('DefaultFieldRendererComponent').textContent).toEqual(
+        'item1,item2,item3,item4,item5  ,+2 More'
+      );
+    });
+  });
+  describe('DefaultFieldRendererOverflow', () => {
+    const idPrefix = 'prefix-1';
+    const rowItems = ['item1', 'item2', 'item3', 'item4', 'item5', 'item6', 'item7'];
+
+    test('it should render the length of items after the overflowIndexStart', () => {
+      render(
+        <TestProviders>
+          <DefaultFieldRendererOverflow
+            idPrefix={idPrefix}
+            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
+            overflowIndexStart={5}
+            rowItems={rowItems}
+            attrName={'mock.attr'}
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('DefaultFieldRendererOverflow-button').textContent).toEqual(
+        '+2 More'
+      );
+      expect(screen.queryByTestId('more-container')).not.toBeInTheDocument();
+    });
+
+    test('it should render the items after overflowIndexStart in the popover', () => {
+      render(
+        <TestProviders>
+          <DefaultFieldRendererOverflow
+            idPrefix={idPrefix}
+            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
+            overflowIndexStart={5}
+            rowItems={rowItems}
+            attrName={'mock.attr'}
+          />
+        </TestProviders>
+      );
+
+      userEvent.click(screen.getByTestId('DefaultFieldRendererOverflow-button'));
+
+      expect(
+        screen.getByText(
+          'You are in a dialog. Press Escape, or tap/click outside the dialog to close.'
```

Comment:
This is probably generally okay, but if we can test based on text we control or a `data-test-subj` value it may make the test less brittle